ui-backend/catalog-ui-oauth/src/main/java/org/codice/ddf/catalog/ui/oauth/app/OAuthApplication.java

```diff
@@ -201,7 +203,11 @@ String processCodeFlow(Response res, String code, Map<String, Object> state) {
 
     JWT idToken = oidcTokens.getIDToken();
     AccessToken accessToken = oidcTokens.getAccessToken();
-    RefreshToken refreshToken = oidcTokens.getRefreshToken();
+    String refreshToken = null;
+
+    if (!isGuest()) {
+      refreshToken = oidcTokens.getRefreshToken().getValue();
```

Comment:
it looks like there is a potential NPE here: https://www.javadoc.io/doc/com.nimbusds/oauth2-oidc-sdk/latest/com/nimbusds/oauth2/sdk/token/Tokens.html#getRefreshToken()